core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -170,7 +170,10 @@
 
 
 
+#ifndef ADDR_SPACE_SIZE
```

Comment:
Since this is ARM specific perhaps it should go into `core/arch/arm/arm.mk` instead?